### PR TITLE
changed schema column names

### LINKED ISSUE #2918: Remove placeholder text for schema editor
The placeholder text appears to be cut off on my screen.

image

We can probably just remove them, since the tooltips next to the "?" buttons on the header have more information about what each of these fields are, anyway.

### PR DESCRIPTION
**Pull request by Levilian**
Fix #2918

### CODE REVIEW

**Review by yiboliu — COMMENTED** [review]
Probably you need to update the test

File: frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js [hunk 1]
```diff
@@ -237,7 +236,7 @@ class SchemaItem extends React.Component<{
                         key={index}
                         style={{ padding: '5', fontSize: '16px', maxWidth: '100' }}
                     >
-                        {header}
+                        {headerText[header]}
```
Comment (epicfaace) [inline frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js:239]: I think we should still keep saying "generalized-path" and "post-processor", as that is the same terminology we use in the docs: https://codalab-worksheets.readthedocs.io/en/latest/Worksheet-Markdown/#schemas

Comment (Levilian) [reply frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js:239]: So we'll simply remove the placeholder text but keep the column titles?

Comment (Levilian) [reply frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js:239]: Actually this issue was brought up last week during the meeting and Percy said it's fine to have two different sets of terminology - this is for clarity's sake to display a more human-readable name on the front end.

**Review by yuqijin — APPROVED** [review]
looks good to me

**Pull request merged**